scapy/arch/__init__.py

```diff
@@ -25,7 +25,7 @@ def str2mac(s):
         from scapy.arch.bpf.core import get_if_raw_addr
 
 def get_if_addr(iff):
-    return socket.inet_ntoa(get_if_raw_addr(iff))
+    return inet_ntop(socket.AF_INET, get_if_raw_addr(iff))
```

Comment:
This is not really related. However, I had a bug with this while testing.
`inet_ntoa` is getting deprecated, so I'm switching them to `inet_ntop` when scapy bug on them